test/powershell/Language/Parser/Ast.Tests.ps1

```diff
@@ -5,20 +5,22 @@ Describe "The SafeGetValue method on AST returns safe values" -Tags "CI" {
         $HtAst = {
             @{ one = 1 }
             }.ast.Find({$args[0] -is $HashtableAstType}, $true)
+        $HtAst | Should Not BeNullOrEmpty
         $HtAst.SafeGetValue().GetType().Name | Should be Hashtable

```

Comment:
I removed "GetType().Name" However, previously we need to check that `$HtAst` is not `Null` to exclude exception.